packages/core/phase2/lib/generateOperations/resultClassHandlers/handleSentence.test.ts

```diff
@@ -113,34 +113,4 @@ describe("handleSentence", () => {
       }
     ])
   })
-
-  it("should return HO200106 when adjudication does not exist", () => {
```

Comment:
I think we still need to keep this test. When `PNCAdjudicationExists` the code returns empty operations. So we need to cover this in this test.